app/indices/base_index.rb

```diff
@@ -44,6 +44,19 @@ def index_name=(value)
       self._index = value
     end
 
+    def search(search_query, model)
```

Comment:
Probably better to do `model` as a keyword parameter, then we can add more options for the `.search` method over time and not deal with refactoring calls or providing backwards compatibility in here